README.md

```diff
@@ -1,222 +1,166 @@
 
-After running setup script, you'll need to create `wp-config.php`. You can do that manually, or use WP-CLI
+    ```php
+    // Must be set.
+    // Possible options are develop, staging and production.
+    define( 'INF_ENV', 'develop' );
 
-```bash
-wp config create --dbname={DBNAME} --dbuser={DBUSER} --dbpass={DBPASS}
-wp core install --url={dev.boilerplate.com} --title={THEMENAME} --admin_user={ADMINUSER} --admin_email={ADMINMAIL}
-wp theme activate {THEMENAME}
-```
+    /* That's all, stop editing! Happy blogging. */
 
-## Development Start
+    /** Absolute path to the WordPress directory. */
+    if ( !defined('ABSPATH') )
+      define('ABSPATH', dirname(__FILE__) . '/');
 
-Builds assets in watch mode using Webpack.
+    // Include wp config for your project.
+    require_once(ABSPATH . 'wp-config-project.php');
 
-```bash
-npm start
-```
+    /** Sets up WordPress vars and included files. */
+    require_once(ABSPATH . 'wp-settings.php');
+    ```
 
-## Browser sync
+    * Remove `define( 'WP_DEBUG', false );`. This is defined in the `wp-config-project.php` file.
 
-We are using BrowserSync to sync assets and enable easy cross-device testing.
-It is tested on MAMP and Vagrant (VVV).
+    * Set `INF_ENV` variable to correspond with your environment. That will add global setup for logging errors, disabling auto-update, some optimizations and it will change the color of admin so it is easier for you to know on what environment you are editing.
 
-## Linting Assets (JS,SASS)
 
-Lints JS and SASS using Webpack
+3. You have BrowserSync to sync assets and enable easy cross-device testing. Once BrowserSync is active you will get url in the terminal. Usualy it is `localhost:3000`.
 
-```bash
-npm run precommit
-```
+## Liniting Files
 
-## Linting PHP ##
+1. **Linting assets** - JS and SASS using Webpack:
 
-We are using [Infinum coding standards for WordPress](https://github.com/infinum/coding-standards-wp) to check php files.
+    ```bash
+    npm run precommit
+    ```
 
-To install it, you need to install [Composer](https://getcomposer.org/) first.
+2. **Linting PHP** - We are using [Infinum coding standards for WordPress](https://github.com/infinum/coding-standards-wp) to check php files. To install it, you need to install [Composer](https://getcomposer.org/) from initial setup step. 
 
-* Add this aliases to you bash config:
+3. **Linting PHP** - If you have composer installed you can add these aliases to your bash/zsh config for easier usage:
 
-```bash
-alias phpcs='vendor/bin/phpcs';
-alias phpcbf='vendor/bin/phpcbf';
-alias wpcs='phpcs --standard=vendor/infinum/coding-standards-wp/Infinum';
-alias wpcbf='phpcbf --standard=vendor/infinum/coding-standards-wp/Infinum';
-```
-* Reload terminal
+    ```bash
+    alias phpcs='vendor/bin/phpcs';
+    alias phpcbf='vendor/bin/phpcbf';
+    alias wpcs='phpcs --standard=vendor/infinum/coding-standards-wp/Infinum';
```

Comment:
In the case where phpcs is run using the vendor version, the standard isn't necessary since it will pick up the `phpcs.xml.dist` as the configuration. In that file we have specified the Infinum's coding standard as a default. So the last two aliases are not really needed 😉

Reply:
ok I will remove it then.